frontend/components/synthesizePolygons/PolygonEditor.module.scss

```diff
@@ -1,25 +1,100 @@
+.tooltipContainer {
+    position: relative;
+    .editorTooltip, .editorTooltipLeft, .editorTooltipRight {
+        visibility: hidden;
+
+        // Position the tooltip
+        position: absolute;
+        z-index: 1;
+        width: 100px;
+        padding: 2px;
+
+        background-color: #d7d7d7;
+        color: #000000;
+        border-style: solid;
+        border-color: black;
+        text-align: center;
+
+        line-height: 1;
+    }
+    .editorTooltip {
+        bottom: 110%;
+        left: 0;
+    }
+    .editorTooltipLeft {
+        bottom: 110%;
+        right: 0;
+    }
+    .editorTooltipRight {
+        left: 120%;
+        top: 0;
+    }
+}
+
+.tooltipContainer:hover, .tooltipDiv:hover {
+    .editorTooltip, .editorTooltipLeft, .editorTooltipRight {
+        visibility: visible;
+    }
+}
+
+.tooltipDiv {
+    border-style: solid;
+    border-color: black;
+    border-width: 1px;
+    border-radius: 50%;
+
+    $size: 15px;
+    width: $size;
+    height: $size;
+
+    div {
+        font-size: 8pt;
+        font-weight: bolder;
+        display: flex;
+        height: 100%;
+        width: 100%;
+        align-items: center;
+        justify-content: center;
+    }
+
+    margin: 0px 1px;
+
+    display: inline-block;
+
+    background-color: #d7d7d7;
+}
+
 .editorContainer {
     display: flex;
     flex-direction: column;
     flex-wrap: nowrap;
     //align-items: center;
+    width: 100%;
+    height: 100%;
 }
 
 .buttonRow {
     display: flex;
     flex-direction: row;
-    flex-wrap: nowrap;
+    justify-content: space-between;
+}
+
+.buttonGroup {
+    display: flex;
+    flex-direction: row;
 }
 
 .editorButton {
     background-color: #007bff;
     border: 1px solid #007bff;
     border-radius: 0.25rem;
     color: white;
-    padding: 0.5rem 0.75rem;
+    padding: 0.5rem 0.5rem;
+    min-height: 40px;
```

Comment:
Might want to add `min-width: 45px;` as Firefox isn't playing nice
<img width="140" alt="Screen Shot 2021-11-05 at 11 03 13 AM" src="https://user-images.githubusercontent.com/1494102/140532088-4411c0d3-2120-482b-bfda-775c17501def.png">